charter.html

```diff
@@ -289,13 +299,22 @@ <h2 id=work-items>Work Items</h2>
 <p>The Chairs may add Work Items, but must not add Work Items which lack
 the support of at least two <a href=#implementer>implementers</a>.
 
-<p class=note>Typically, Work Items begin life as a Proposal before
-being adopted.
+<p>Each Work Item should be accompanied by an explainer describing its
+proposed changes to the web platform. Editors should keep the Work
+Item's explainer up-to-date with the Work Item as it evolves.
+
+<p class=note>Since Work Items typically begin life as a Proposal before
+being formally adopted, they usually start out with an explainer.
+
+<p>When a Work Item's Editors deem the Work Item ready for migration,
+they will notify the Chairs. The CG may produce a Final Community Group
+Report at this time. The Editors and Chairs will identify the best
+destination standards body or bodies, and will then work with those
+bodies to successfully migrate the Work Item.
 
-<p>When a Work Item's Editors deem the Work Item ready for migration to
-a standards body, they will notify the Chairs. The Editors and Chairs
-will then work together to identify the best destination, and will work
-with the destination standards body to successfully migrate the work.
+<p class=note>When migrated to the standards track, Work Items might
+become standalone specifications, they might be integrated into one or
+more existing specifications, or even some combination of these options.
```

Comment:
Grammar nit: these three aren't parallel: the first two have subjects and verbs (Work Items might become, they might be integrated), while the third is just an object.

```suggestion
more existing specifications, or we might even pick some combination of these options.
```

Reply:
Fixed in ca761c1.